packages/allure-js-commons/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "include": [
     "./src/**/*",
+    "./test/**/*",
```

Comment:
This line adds tests code to the final npm artifacts. This is not happening in other packages